masu/config.py

```diff
@@ -119,8 +119,11 @@ class Config:
     # Insights Kafka messaging address
     INSIGHTS_KAFKA_HOST = os.getenv('INSIGHTS_KAFKA_HOST', 'localhost')
 
-    # Insights Kafka messaging address
+    # Insights Kafka messaging port
     INSIGHTS_KAFKA_PORT = os.getenv('INSIGHTS_KAFKA_PORT', '29092')
 
+    # Insights Kafka connection retry interval
+    INSIGHTS_KAFKA_CONN_RETRY_INTERVAL = int(os.getenv('INSIGHTS_KAFKA_CONN_RETRY_INTERVAL', '30'))
```

Comment:
should we try/catch here, in case someone mistakenly gives `INSIGHTS_KAFKA_CONN_RETRY_INTERVAL` something that cannot be converted to an int?